scripts/test-check-packages.sh

```diff
@@ -38,7 +38,7 @@ OLDPWD=$PWD
 for d in test/packages/${PACKAGE_TEST_TYPE:-other}/${PACKAGE_UNDER_TEST:-*}/; do
   (
     cd $d
-    elastic-package check -v
+    elastic-package build -v
```

Comment:
Actually, the `check` commands runs also `format` and `validate`. This test script is to verify the `check` call. Is there an option to revert this change?

Reply:
this slipped through 🙏